Puzzle/src/org/fundationjala/devops/bootcamp/main/java/Position.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class Position {
+
+    private int row;
+    private int column;
+
+    public Position(int row, int column) {
+        this.row = row;
+        this.column = column;
+    }
+
+    public Position() {
+
+    }
+
+    public int getRow() {
+        return row;
+    }
+
+    public void setRow(int row) {
+        this.row = row;
+    }
+
+    public int getColumn() {
+        return column;
+    }
+
+    public void setColumn(int column) {
+        this.column = column;
+    }
+
+    public Position enterPosition (){
```

Comment:
I think you can have a class for asking for input data from the console because I think Position should not take care of asking a position, right?